scripts/fix-links.js

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env node
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+'use strict';
+
+const fs = require('fs');
+const { platform } = require('os');
+
+const { processData } = require('../test/linter/test-links.js');
+
+/** Determines if the OS is Windows */
+const IS_WINDOWS = platform() === 'win32';
+
+/**
+ * @param {Promise<void>} filename
```

Comment:
Based on the following, filename is a `string`:

https://github.com/mdn/browser-compat-data/blob/1d4f83058631578fd298b759cfc7ae1cb90c9655/test/linter/test-links.js#L11-L17

```suggestion
 * @param {string} filename
```